### PR TITLE
refactor: move cvtask from PredictModel method to CreateModel method

### PR DESCRIPTION
**Pull request by Phelan164**
Because

- User know CV Task when creating a model, then it is convenient to add CV Task at that point

This commit

- Move CV Task from PredictModel method to CreateModel method

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,15 +20,13 @@ require (
 	go.uber.org/zap v1.20.0
 	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
 	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
-	google.golang.org/genproto v0.0.0-20220211171837-173942840c17 // indirect
+	google.golang.org/genproto v0.0.0-20220215190005-e57b466719ef // indirect
```
Comment (bochengyang) [inline go.mod:23]: This looks like a up-to-date version, the latest commit SHA should be `2d9577e0f42b14380cd86ddd1d9ea1a65e85ef18`

File: rpc/model.go [hunk 2]
```diff
@@ -110,57 +114,51 @@ func unzip(filePath string, dstDir string) ([]*models.Model, []*models.Version,
 					sVersion := elems[len(elems)-1]
 					iVersion, err := strconv.Atoi(sVersion)
 					if err == nil {
-						newVersion := &models.Version{
+						currentModel.Versions = append(currentModel.Versions, models.VersionResponse{
 							Version:   int32(iVersion),
```
Comment (bochengyang) [inline rpc/model.go:118]: Looks like there is a high level warning from this line

File: go.mod [hunk 3]
```diff
@@ -20,14 +20,14 @@ require (
...
```
Comment (bochengyang) [inline go.mod:23]: This should be `github.com/instill-ai/protogen-go v0.0.0-20220216035831-2d9577e0f42b`

File: go.mod [hunk 4]
```diff
@@ -20,14 +20,14 @@ require (
...
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
 	gorm.io/driver/postgres v1.2.3
 	gorm.io/gorm v1.22.5
 )
 
-require github.com/instill-ai/protogen-go v0.0.0-20220214084110-9a2713d704b2
+require github.com/instill-ai/protogen-go v0.0.0-20220216035831-2d9577e0f42b
```
Comment (bochengyang) [inline go.mod:30]: This line should remove

**Comment by Phelan164** [comment]
@bochengyang yes, it warning about casting  strconv.Atoi output int to int32 which maybe problem in some hardward spec. But IMO version is small number and casting have no problem

**Comment by bochengyang** [comment]
> @bochengyang yes, it warning about casting strconv.Atoi output int to int32 which maybe problem in some hardward spec. But IMO version is small number and casting have no problem

Is there any possibility to have the same type of conversion?

**Comment by Phelan164** [comment]
@bochengyang about requires in go.mod, I used go mod tidy and do not manually modify that.

**Comment by bochengyang** [comment]
> @bochengyang about requires in go.mod, I used go mod tidy and do not manually modify that.

It is weird because we only have two parts of the `require` section. How about you delete the middle one and do it again by following the command

```bash
go get github.com/instill-ai/protogen-go@v0.0.0-20220216035831-2d9577e0f42b
```

**Comment by Phelan164** [comment]
@bochengyang yes, I tried go get {package} it more clean. Do not know why go mod tidy like that. I updated

File: go.mod (hunk 3 above)
Comment (bochengyang) [reply go.mod:23]: The version is still not the up-to-date one

Comment (bochengyang) [reply go.mod:23]: oh, my mistake, that's not our package

**Review by bochengyang — APPROVED** [review]
LGTM 🚀

**Pull request merged**